tools/Resources.py

```diff
@@ -240,24 +245,29 @@ def get_latest_number_properties_multiple(target, token, uuids, propkey):
         properties_list = list()
 
         if response.status_code == 200:
-            if not response.json()['values']:
-                print("skipping propkey " + str(propkey) + ", no return")
-                return False
-            for resource in response.json()['values']:
-                d = dict()
-                d['resourceId'] = resource['resourceId']
-                d['propkey'] = propkey
-                content = resource['property-contents']['property-content']
-                if content:
-                    if 'values' in content[0]:
-                        d['data'] = content[0]['values'][0]
+            try:
+                if not response.json()['values']:
```

Comment:
I'd say we only try catch this

Reply:
without catching JSONDecodeError and message?

Reply:
The problem always occurs here with `['values']`, so why take in the other stuff too?

Reply:
got it